src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/UserPersonDetail.kt

```diff
@@ -183,13 +190,14 @@ data class UserPersonDetail(
   }
 
   private fun addUserGroup(userGroup: UserGroup) {
-    activeAndInactiveMemberOfUserGroups.firstOrNull { it.id.userGroupCode == userGroup.id && it.id.username == this.username } ?: run {
-      val member = UserGroupMember(
-        id = UserGroupMemberPk(userGroupCode = userGroup.id, username = this.username),
-        user = this, userGroup = userGroup
-      )
-      activeAndInactiveMemberOfUserGroups.add(member)
-    }
+    activeAndInactiveMemberOfUserGroups.firstOrNull { it.id.userGroupCode == userGroup.id && it.id.username == this.username }
```

Comment:
This is confusing as these are not actual changes

Reply:
Format change